includes/scrub-options.php

```diff
@@ -22,7 +22,7 @@ function scrub_options() {
 		$option_value = get_option( $option );
 		if ( $option_value ) {
 
-			update_option( $option . '_backup', $option_value );
+			update_option( $option . '_sn_backup', $option_value );
```

Comment:
I left this call to `update_option()` in place, because we're creating this option, so it's just as (if not more) likely  to be adding the option instead of updating it. To be more careful, I changed the backup option name to indicate it's Safety Net, in case the owning plugin/theme has its own `_backup` options that could trigger filters.

Reply:
This is definitely a good idea